spec/adapters/postmark_adapter/outbound_spec.rb

```diff
@@ -74,6 +74,14 @@
 
         subject { message_email[:from].formatted }
         it { should eq(['TestingProject <100eyes-test-account@example.org>']) }
```

Comment:
I was wondering why this test doesn’t fail with that change? Shouldn’t this be `"TestingProject" <100eyes-test-account@example.org>` (with quotes)?

Reply:
Never mind, I didn’t notice that this doesn’t test for the raw `From` header, but for the `formatted` value, which parses, then converts back to a string again, which only adds quotes if needed.

https://github.com/mikel/mail/blob/22a7afc23f253319965bf9228a0a430eec94e06d/lib/mail/elements/address.rb#L47